migration/ods1_stage/tables/base/address/spu_translated_address.sql

```diff
@@ -0,0 +1,96 @@
+--- 1. etl_spuMergeOfficeAddress (snowflake db)
+
+-- CTE 'w' retrieves records from FacilityProfileProcessing and FacilityProfileProcessingDeDup tables
+-- It also extracts 'Address' from 'EntityJSONString' and assigns it to the 'FacilityJSON' column
+WITH cte_w AS (
```

Comment:
We should use more self-described aliases.
For example, here could be cte_facility